pkg/tfbridge/provider_test.go

```diff
@@ -4738,7 +4742,8 @@ func TestUnknowns(t *testing.T) {
 		},
 		"response": {
 			"properties":{
-				"id":""
+				"id":"",
+                                "setBlockProps": [{"prop": ""}]
```

Comment:
This looks like an improvement but still not quite what we want. We had a set of 1 element `[unk]` and now it shows  back as indeed a set of one element, just the unknonwn got substituted by an empty value.

@VenelinMartinov points out this may be fixed/changed in https://github.com/pulumi/pulumi-terraform-bridge/pull/2060